app/helpers/application_helper.rb

```diff
@@ -161,10 +161,18 @@ def display(attribute)
     "display:#{attribute ? 'none' : 'inline'};"
   end
 
+  def controller_namespace
```

Comment:
These changes might not be needed if the import action is implemented on `ComputeResourcesVmsController`